src/coreclr/src/tools/Common/TypeSystem/Common/ExceptionStringID.cs

```diff
@@ -34,6 +34,7 @@ public enum ExceptionStringID
         InvalidProgramNativeCallable,
         InvalidProgramCallAbstractMethod,
         InvalidProgramCallVirtStatic,
+        InvalidProgramNativeCallableAttributeUsage,
```

Comment:
Nit: Could you add IDs for all 3 variants of these messages? While warnings from crossgen are not super important (but we will want to surface them), the type system is eventually going to be used by the ILVerify tool once it's moved over (#13827) and it shouldn't be that we get more descriptive error message from the runtime than from the tool.